Source/SwiftLintFramework/Rules/FirebaseConfigDefaultsRule.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FirebaseConfigDefaultsRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigDefaultsRule: ConfigurationProviderRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_defaults",
+        name: "Firebase Config Defaults",
+        description: "Firebase Config defaults should be set.",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.setDefaults(fromPlist: \"RemoteConfigDefaults\")\n" +
+            "  }\n" +
+            "}"
+        ],
+        triggeringExamples: [
+            "class ViewController: UIViewController {\n" +
```

Comment:
For instance, this rule triggered a violation on one of [Firefox's View Controllers](https://github.com/mozilla-mobile/firefox-ios/blob/067b6670e8216e7822f94dd3881f25ff481d31b8/Extensions/SendTo/ActionViewController.swift#L19:14). If Firefox were using Firebase, does it mean this View Controller would have to call `.setDefaults(fromPlist:)`?